### PR TITLE
Challenges of Getting Started translated to Indonesia

### LINKED ISSUE #8025: Indonesian Translation of Getting Started
Because there's  no Indonesian translation existed, so I want to start one.

### PR DESCRIPTION
**Pull request by tamami**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [x] Add new translation (feature adding new translations)
#### Checklist:

- [ ] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #8025 
#### Description

Add Indonesia Translation for Getting Started.

### CODE REVIEW

**Comment by raisedadead** [comment]
@tamami thanks for the pull request, can we have someone to do a peer review. We can merge post that.

**Comment by alistermada** [comment]
Just want to point out that this translation is only for the first challenge of Getting Started.

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -85,6 +85,81 @@
       "tests": [],
       "type": "Waypoint",
       "challengeType": 7,
+      "titleId": "Mengetahui Bagaimana Free Code Camp bekerja",
+      "descriptionId": [
+	[
+	  "//i.imgur.com/6ibIavQ.jpg",
+	  "Gambaran 4 kelebihan Free Code Camp: Terhubung, Belajar JavaScript, Membangun Portofolio, Membantu nirlaba",
```
Comment (alistermada) [inline seed/challenges/00-getting-started/getting-started.json]: Wouldn't "nonprofits" be better translated to _organisasi nirlaba_? Can you use _nirlaba_ as a noun?

File: seed/challenges/00-getting-started/getting-started.json [hunk 2]
```diff
@@ -85,6 +85,81 @@
...
+	],
+	[
+	  "//i.imgur.com/WD3STY6.jpg",
+	  "Foto 3 campers yang mendapatkan pekerjaan setelah belajar code di Free Code Camp.",
+	  "Free Code Camp adalah jalan ampuh menjadi developer software. Banyak perusahaan mempekerjakan ribuan orang seperti kamu yang belajar code melalui Free Code Camp.",
+	  ""
+	],
+	[
+	  "//i.imgur.com/D7Y5luw.jpg",
+	  "Grafik rata rata pertumbuhan pekerjaan berdasarkan pertumbuhan lulusan ilmu komputer. Tersedia 1.4 juta pekerjaan dan hanya 400 ribu orang yang mengisi tempatnya.",
```
Comment (alistermada) [inline seed/challenges/00-getting-started/getting-started.json]: The growth of jobs is compared to the growth of computer science graduates in the graph. I think _dibanding_ would be more accurate than _berdasarkan_

Change 1.4 to 1,4.

File: seed/challenges/00-getting-started/getting-started.json (hunk 1 above)
Comment (tamami) [inline seed/challenges/00-getting-started/getting-started.json]: owh.. ya.. ok.. :+1:

File: seed/challenges/00-getting-started/getting-started.json (hunk 2 above)
Comment (tamami) [inline seed/challenges/00-getting-started/getting-started.json]: ok..

**Comment by raisedadead** [comment]
Awesome @alistermada thanks for QA. Letvus know when this is ready.

**Comment by alistermada** [comment]
@raisedadead I'm unsure about the usage of some untranslated terms like project, code/coders/coding, and software development. I can confirm that translations are accurate, although the translation for "user story" is a bit iffy. 

I'll ask other Indonesians to help review. We haven't yet established a set of rules to how tech terms will be translated.

**Comment by raisedadead** [comment]
Okay, thanks for the update @alistermada just let us know when this is ready to be merged. Blocking meanwhile.

**Comment by BerkeleyTrue** [comment]
Closing as stale. Please resubmit when ready.

Happy Coding

**Pull request closed without merging**